frontend/src/App.css

```diff
@@ -63,10 +63,32 @@ body {
   justify-content: space-between;
   align-items: center;
   padding: 1rem 0;
+  gap: 2rem;
+}
+
+/* logo container to take more space */
+.header-content > div:first-child {
+  flex: 0 0 auto;
+  min-width: 200px;
+  padding-left: 0.5rem;
+  flex-grow: 1;
+}
+
+/* Navigation container */
+.header-content > div:last-child {
+  min-width: 200px;
+  padding-right: 1rem;
+}
```

Comment:
It's usually best to avoid CSS rules that target a specific HTML structure, like the first and last div children of an element as seen here. That does sort of make sense for adding left and right padding, but not for content-dependent changes like flex settings (we're allowing the first div to grow because we know we're adding more content inside it). Instead, we can add meaningful class names to these divs to reference in the CSS.